src/components/shared/pick-track/index.tsx

```diff
@@ -140,21 +149,81 @@ class PickTrack extends React.Component<IPickTrackProps, IPickTrackState> {
     }
   }
 
+  componentDidMount() {
+    const firebaseApp = import('firebase/app');
+    const firebaseDatabase = import('firebase/firestore');
+    var currentTime = new Date();
+
+    Promise.all([firebaseApp, firebaseDatabase]).then(([firebase]) => {
+      const database = getFirebase(firebase).firestore();
```

Comment:
I'm not sure this is how we should do it, but if it works - great. I'll probably experiment with a few different approaches and make a PR if I find something more suitable

Reply:
I followed it from the link that I gave you. I tried to remove this promise.all but couldn't make it work somehow but as of now with this it works